luaui/Widgets/gui_territorial_domination.lua

```diff
@@ -1622,8 +1641,8 @@ function widget:Update(deltaTime)
 		updateDisplayLists()
 		
 		-- Hide tooltip when major state changes
-		if WG['tooltip'] and WG['tooltip'].HideTooltip then
-			WG['tooltip'].HideTooltip('territorialDomination')
+		if WG['tooltip'] and WG['tooltip'].RemoveTooltip then
```

Comment:
```suggestion
		if WG['tooltip'] then
```
It is sufficient to check the existence of `WG['tooltip]`, if it exists, then so will `WG['tooltip].RemoveTooltip`

Reply:
OK, done